.github/upload-to-s3.sh

```diff
@@ -68,7 +68,7 @@ function uploadFile() {
   uploadFile $FILENAME_SRC $FILENAME_DST $DRY_RUN
   copyOrDownloadToTemp $FILENAME_SRC "$FILENAME_DST" $DRY_RUN
 
-  FILENAME_SRC="$SCRIPT_DIR/../build/snyk-ls_darwin_arm64/snyk-ls"
+  FILENAME_SRC="$SCRIPT_DIR/../build/snyk-ls_darwin_arm64_v8.0/snyk-ls"
```

Comment:
this is because the GH action is now using a new goreleaser 
v2.4.2 instead of v2.3.2